data.sql

```diff
@@ -35,4 +35,18 @@ INSERT INTO animals(
 VALUES(
 	'Devimon', '2017-05-12',
 	'5', 'yes', '11'
-)
\ No newline at end of file
+)
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228251323-0cba88a3-a519-4c8e-a603-4a68b22d7813.png)

**[OPTIONAL]**

- [ ] It would be nice if you fix your MIT link to make your project licensed and readme descriptive :+1: